examples/lib-samples/backend/mindspore/ResNet50/README.md

```diff
@@ -1,70 +1,100 @@
 # Resnet Example with Mindspore Backend
-This document describes how to use the mindspore backend to train Resnet-50 network with the cifar-10 dataset
+This document describes how to use the mindspore backend to train Resnet-50 network with the cifar-10 dataset.
+
+## Script Description
+
+### Script and Sample Code
+```shell
+└──ResNet50
+  ├── README.md
+  ├── scripts
+    ├── run_eval.sh                        # launch ascend evaluation
+    ├── run_eval_cpu.sh                    # launch cpu evaluation
+    ├── run_infer.sh                       # launch cpu inference
+    ├── run_standalone_train.sh            # launch ascend standalone training
+    ├── run_standalone_train_cpu.sh        # launch cpu training
+  ├── src
+    ├── config.py                          # parameter configuration
+    ├── dataset.py                         # data preprocessing
```

Comment:
should we move the way of data preprocessing for midnspore to sedna lib,  instead of in examples?  the other people may be want to use it when they develop a appliation based on mindspore.

Reply:
Where do you think is the most appropriate ? `sedna.datasource` or `sedna.backend.mindspore` ?

Reply:
Should we develop preprocessing methods **for each model**, or develop a **general** preprocessing method?
In my assumption, for a certain scene (such as image classification), we can predefine several fixed preprocessing methods (such as normalize, resize), and the user only needs to pass some parameters.

Reply:
I suggest not to integrate data preprocessing functions into Sedna Lib, for the reason:
- There are various preprocessing functions, and they cannot be covered by Sedna.
- The data preprocessing functions are always already defined in developer's training code or AI framework.